src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Utilities/Contract.cs

```diff
@@ -48,7 +87,7 @@ internal static class Contract
         /// all builds
         /// </summary>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public static void ThrowIfNull<T>([NotNull] T value, string message, [CallerLineNumber] int lineNumber = 0)
+        public static void ThrowIfNull<T>([NotNull] T value, string message, [CallerLineNumber] int lineNumber = 0) // TODO: Interpolated string handler.
```

Comment:
You'll want a `ThrowIfNullHandler<T>` that takes a `T` and `out`'s a `bool` for this one.